src/BlockTypes/ProductQuery.php

```diff
@@ -113,6 +113,22 @@ public function update_query( $pre_render, $parsed_block ) {
 		}
 	}
 
+	/**
+	 * Merge tax_queries from various queries.
+	 *
+	 * @param array[] ...$queries Query arrays to be merged.
+	 * @return array
+	 */
+	private function merge_tax_queries( ...$queries ) {
```

Comment:
This function works perfectly fine as is, but this is how I would have probably written it 👇 (I usually prefer to use `[]` instead of `array()`).
```php
private function merge_tax_queries( ...$queries ) {
	$tax_query = [];
	foreach ( $queries as $query ) {
		$tax_query[] = $query['tax_query'] ?: [];
	}
	return [ 'tax_query' => $tax_query ];
}
```
Let me know if something doesn't make sense to you 🙏

Reply:
I think there's a difference between these two implementations (and the tests caught that! 😄). `$tax_query[] = $query['tax_query'] ?: [];` adds an empty array to the `$tax_query` array. So before returning, `$tax_query` would have to be filtered out. Unless I'm not getting something 😅 

I kept the original implementation for now 🙌

Reply:
Hey, sorry! That's what happens when you try to be too clever on a Friday and don't try your code ✌️
You are right, the code it's not equivalent, better to keep the original! 😅
I think outside tests you can use the `[]`